tests/dags/providers/provider_api_scripts/resources/finnishmuseums/object_complete_example.json

```diff
@@ -1,54 +1,24 @@
 {
+  "authors": {
+    "primary": { "Uomala, valokuvaaja": { "role": ["-"] } },
+    "secondary": [],
+    "corporate": []
+  },
   "buildings": [
-    {
-      "translated": "Museovirasto",
-      "value": "0/Museovirasto/"
-    },
-    {
-      "translated": "Kansatieteen kuvakokoelma",
-      "value": "1/Museovirasto/Kansatieteen kuvakokoelma/"
-    }
-  ],
-  "formats": [
-    {
-      "translated": "Kuva",
-      "value": "0/Image/"
-    },
-    {
-      "translated": "Kuva",
-      "value": "1/Image/Image/"
-    }
+    { "value": "0/SA-kuva/", "translated": "Sotamuseo" },
+    { "value": "1/SA-kuva/SA-kuva/", "translated": "SA-kuva-arkisto" }
   ],
-  "id": "museovirasto.CC0641BB5337F541CBD19169838BAC1F",
+  "id": "sa-kuva.sa-kuva-1835",
   "imageRights": {
     "copyright": "CC BY 4.0",
+    "link": "http://creativecommons.org/licenses/by/4.0/deed.fi",
     "description": [
-      "Kuva on julkaistu Nime\u00e4 4.0 Kansainv\u00e4linen (CC BY 4.0)-lisenssill\u00e4. Verkkopalvelussa julkaistua n\u00e4ytt\u00f6kuvaa voi k\u00e4ytt\u00e4\u00e4 vapaasti my\u00f6s kaupallisessa k\u00e4yt\u00f6ss\u00e4. Kuvan yhteydess\u00e4 on mainittava tekij\u00e4 ja/tai kuvaaja sek\u00e4 organisaation ja kokoelman nimi. Kuvan k\u00e4ytt\u00e4j\u00e4ll\u00e4 on vastuu tekij\u00e4noikeuksien ja yksityisyyden suojan kunnioittamisesta. Esimerkiksi henkil\u00f6kuvien k\u00e4ytt\u00f6 markkinoinnissa ja/tai mainonnassa on kielletty ilman kuvassa olevan henkil\u00f6n suostumusta. Museovirasto pyrkii julkaisemansa tiedon oikeellisuuteen. Kuvien tiedoissa voi kuitenkin olla puutteellisuuksia ja virheit\u00e4. Virheen huomatessanne pyyd\u00e4mme ottamaan yhteytt\u00e4 Kuvakokoelmien asiakaspalveluun. Kuvia ja tietoja julkaistaessa asiakkaan vastuulla on varmistaa niiden oikeellisuus. Painolaatuiset kuvat ovat maksullisia. Lis\u00e4tietoja tarvittaessa: Museoviraston Kuvakokoelmat (kuvakokoelmat@museovirasto.fi)."
-    ],
-    "link": "http://creativecommons.org/licenses/by/4.0/deed.fi"
+      "SA-kuvat ovat lisensoitu Nime\u00e4 4.0 Kansainv\u00e4linen (CC BY 4.0) -lisenssill\u00e4 ja ne ovat vapaasti k\u00e4ytett\u00e4viss\u00e4 ja julkaistavissa. Kuvat ovat n\u00e4ht\u00e4viss\u00e4 my\u00f6s www.sa-kuva.fi -palvelussa."
```

Comment:
It seems that `ensureAscii=false` wasn't used when saving the `json` file so that the utf-8 characters are not converted to `\uxxx`-type codes.